### PR TITLE
[INTEL_MKL] Reverting bias cache optimization

### PR DESCRIPTION
**Pull request by Srini511**
Reverting bias cache optimization because of some internal failures

### CODE REVIEW

File: tensorflow/core/kernels/mkl_conv_ops.cc [hunk 1]
```diff
@@ -1587,6 +1585,10 @@ class MklQuantizedConv2DOp
       // Re-scale bias if either of following 2 conditions are met:
       // 1. Bias is not const;
       // 2. Bias is const, but bias cache is empty (first iteration).
+      // TODO Disabling bias caching for now. The graph obtained after
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl_conv_ops.cc]: Nit: Please add one blank line to separate this from the normal comment above.
Nit: TODO should be an action item. Maybe rephrase it?
```c++
// TODO(intel-tf): Re-enable bias caching. Currently, the graph obtained 
// after quantize_graph.py does not run with correct accuracy with this 
// feature enabled.
```

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for the PR!

File: tensorflow/core/kernels/mkl_conv_ops.cc [hunk 2]
```diff
@@ -1585,9 +1585,11 @@ class MklQuantizedConv2DOp
...
       // 1. Bias is not const;
       // 2. Bias is const, but bias cache is empty (first iteration).
-      // TODO Disabling bias caching for now. The graph obtained after
-      // quantize_graph.py does not run with correct accuracy when this feature
-      // is enabled.
+
+      // TODO(intel-tf): Re-enable bias caching. Currently, the graph obtained
+      // after quantize_graph.py does not run with correct accuracy with this
+      // feature enabled.
+
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl_conv_ops.cc]: Nit: Please remove this blank line. I don't think we need this extra space between comment and code. (I only needed the blank line in line 1588 but not 1592.) Sorry about the nit!

**Comment by Srini511** [comment]
@penpornk please take a look

**Review by penpornk — CHANGES_REQUESTED** [review]
Thanks so much for your patience!

**Comment by Srini511** [comment]
Hi @penpornk , i think i have made the changes requested. :-)

**Review by penpornk — APPROVED** [review]
Oops, sorry! I meant to click approve. :)
